plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -154,6 +183,10 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
     }
 
     onBindingClick = async (binding: AppBinding) => {
```

Comment:
Apart of not allowing to hit the buttons, do we want any visual feedback of they become unavailable? Or not? (I guess not, but wanted to ask the question just in case)